src/cascadia/TerminalSettingsModel/AppearanceConfig.cpp

```diff
@@ -172,20 +172,20 @@ winrt::hstring AppearanceConfig::ExpandedBackgroundImagePath()
     }
 }
 
-void AppearanceConfig::_logSettingSet(std::string_view setting)
+void AppearanceConfig::_logSettingSet(const std::string& setting)
 {
     _changeLog.emplace(setting);
 }
 
-void AppearanceConfig::_logSettingIfSet(std::string_view setting, const bool isSet)
+void AppearanceConfig::_logSettingIfSet(const std::string_view& setting, const bool isSet)
 {
     if (isSet)
     {
-        _logSettingSet(setting);
+        _logSettingSet(std::string{ setting });
     }
 }
 
-void AppearanceConfig::LogSettingChanges(std::set<std::string_view>& changes, std::string_view& context) const
+void AppearanceConfig::LogSettingChanges(std::set<std::string>& changes, const std::string& context) const

```

Comment:
what I said about leaking implementation details here applies to `context` as well. It doesn't **NEED** to be a `string`. It can be a string_view - otherwise, you are risking callers making a copy of a string to pass to `LogSettingChanges` _and then immediately making another copy on line R192 when you format with it_.